### ISSUE TITLE
[General] Limitations of Chakra UI

### ISSUE THREAD

**Issue by adrianoresende** [chakra-ui/chakra-ui#1463]
Hello there,

I was reading here and I saw that this article not is mentioned here.

https://whoisryosuke.com/blog/2020/limitations-of-chakra-ui/

I would like to hear about it, I don't know if the UI is so themeable, but it looks like there are some limitations that could be an issue for us in the future? WDYT?

**Comment by segunadebayo** [comment]
Hi @adrianoresende,

Yes, I saw that article and the concerns there are valid if you're using Chakra v0.8.

We tried to solve these issues in v1 and I believe literally every component in Chakra v1 is themeable and customizable.

We created a theming API that looks something like this:
https://github.com/chakra-ui/chakra-ui/blob/develop/website/docs/theming/overview.mdx
https://github.com/chakra-ui/chakra-ui/blob/develop/website/docs/theming/basic.mdx

We also ensured all components follow this pattern to make it easy to customize. here's an example:
https://github.com/chakra-ui/chakra-ui/blob/develop/packages/theme/src/components/button.ts

V1 is currently in `rc` and would be official in the coming weeks. I hope you get the idea and resolved any doubt you have for the future. We're committed to making Chakra UI components as accessible, composable, and performant as possible.

Cheers.
